pkg/accounting/accounting.go

```diff
@@ -1159,7 +1159,7 @@ func (a *Accounting) NotifyRefreshmentSent(peer swarm.Address, attemptedAmount,
 	// compare received refreshment amount to expectation
 	if expectedAllowance.Cmp(amount) > 0 {
 		// if expectation is not met, blocklist peer
-		a.logger.Error(ErrEnforceRefresh, "pseudosettle peer", peer, "accepted lower payment than expected")
+		a.logger.Error(nil, "allowance expectation refused", "pseudosettle peer", peer)
```

Comment:
why can't we go with "accepted lower payment than expected"

Reply:
but if it is not an error, and if error field is not being used, then why use Error in first place

Reply:
go with 
`a.logger.Error(nil, "accepted lower payment than expected", "pseudosettle_peer", peer)`

Reply:
> go with `a.logger.Error(nil, "accepted lower payment than expected", "pseudosettle_peer", peer)`

sure but 2 lines below you are blocklisting a peer for a different message so just putting this here

Reply:
the intention here is that with the Error func it has a highest logging level,  so if a user turns off Debug or warning level logs, this error msg won't show.

And we go with a nil error because it was not an error we "caught" from a function call or something

Reply:
yes then it might not be an error. but anyways, ive made the changes